tests/unit/lastuser_core/test_model_user_User.py

```diff
@@ -0,0 +1,466 @@
+        kate_password = kate._set_password(u'12thprecinct')
+        db.session.add(kate)
+        db.session.commit()
+        result = models.User.get(userid=kate.userid)
+        self.assertEqual(len(result.pw_hash), 60)
+        self.assertGreater(result.pw_expires_at, result.pw_set_at)
+
+    def test_user_clients_with_team_access(self):
+        """
+        Test to verify a list of clients with access to the user's organizations' teams.
+        """
+        aro = models.User(username=u'aro')
+        jane = models.User(username=u'jane')
+        marcus = models.User(username=u'marcus')
+        volturi = models.Organization(name=u'volturi', title=u'The Volturi')
+        volturi.owners.users.append(aro)
+        volturi.members.users.append(marcus)
+        volturi.members.users.append(jane)
+        volturi.make_teams()
+        volterra = models.Client(title=u'Volterra, Tuscany', org=volturi, confidential=True, website=u'volterra.co.it')
+        enforcers = models.Client(title=u'Volturi\'s thugs', org=volturi, confidential=True, website=u'volturi.co.it')
+        volterra_auth_token = models.AuthToken(client=volterra, user=aro, scope=u'teams', validity=0)
+        enforcers_auth_token = models.AuthToken(client=enforcers, user=marcus, scope=u'teams', validity=0)
+        self.assertItemsEqual(aro.clients_with_team_access(), [volterra])
+        self.assertItemsEqual(marcus.clients_with_team_access(), [enforcers])
+        self.assertEqual(jane.clients_with_team_access(), [])
+
+    def test_user_password_has_expired(self):
+        """
+        Test to check if password for a user has expired
+        """
+        alexis = models.User(username=u'alexis', fullname=u'Alexis Castle')
+        alexis._set_password(u'unfortunateincidents')
+        alexis.pw_expires_at=datetime.utcnow()+ timedelta(0,0,1)
+        db.session.add(alexis)
+        db.session.commit()
+        result = models.User.get(userid=alexis.userid)
+        self.assertTrue(alexis.password_has_expired())
+
+    def test_user_password_is(self):
+        """
+        Test to retrieve hashed password for a user
+        """
+        # scenario 1: no password been set
+        oldmajor = models.User(username=u'oldmajor')
+        self.assertFalse(oldmajor.password_is(u'oinkoink'))
+        # scenario 3: if password has been set
+        dumbeldore = models.User(u'dumbeldore', fullname=u'Albus Dumberldore')
+        dumbeldore_password=u'dissendium'
+        dumbeldore._set_password(password=dumbeldore_password)
+        self.assertTrue(dumbeldore.password_is(dumbeldore_password))
+
+    def test_user_is_active(self):
+        """
+        Test for user's ACTIVE status
+        where ACTIVE = 0 indicates a Regular, active user
+        """
+        crusoe = self.fixtures.crusoe
+        self.assertEqual(crusoe.status, 0)
+        oakley = models.User.query.filter_by(username=u"oakley").first()
+        oakley.status = 1
```

Comment:
user status is marked as active by default, and that's what you should be testing. You shouldn't have to do this assignment.